### DISCUSSION TITLE
M5Stack PoEsp32 for Faikin?

### DISCUSSION THREAD

**Discussion by BeFygo** [revk/xESP32-Faikout#387] [Q&A]
Hi,

Would it be possible to wire the uart from the  M5Stack PoEsp32 to a s21 or A50X port and run Faikin?

The m5stack would get it's power from POE.

https://docs.m5stack.com/en/unit/poesp32

**Comment by revk** [comment]
Probably. It is just serial.

The main issue is that S21 uses 5V signalling, and 3.3V does not quite work on all models, so a level convertor may be needed.
